python/test/test_hail/expr/test_expr.py

```diff
@@ -272,6 +272,15 @@ def test_aggregators_linreg(self):
         self.assertAlmostEqual(r.p_value[1], 0.63310173)
         self.assertAlmostEqual(r.n, 5)
 
+    def test_aggregators_virtualds(self):
+        xs = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        ys = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        table = hl.Table.parallelize([hl.struct(x=x, y=y) for (x, y) in zip(xs, ys)])
+        r = table.aggregate(agg.virtualds(table.x, table.y, 0, 10, 0, 10, 5))
+        xs = [x for (x, y) in r]
+        ys = [y for (x, y) in r]
+        self.assertTrue(set(xs) == set([1.0, 3.0, 5.0, 7.0, 9.0]) and set(ys) == set([1.0, 3.0, 5.0, 7.0, 9.0]))
```

Comment:
Instead of comparing the xs and the ys together, you might compare them as pairs, in case the wrongs xs are being returned with the wrong ys.f